http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java

```diff
@@ -116,6 +122,7 @@
     private static final Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(2);
     private static final Duration DEFAULT_CONNECTION_ACQUIRE_TIMEOUT = Duration.ofSeconds(10);
     private static final Duration DEFAULT_CONNECTION_MAX_IDLE_TIMEOUT = Duration.ofSeconds(60);
+    private static final Boolean DEFAULT_CONNECTION_KEEPALIVE = Boolean.TRUE;
```

Comment:
We should probably make it default to false so that it has the same behavior as now.

Reply:
Sorry, thought it was, not sure what happened there.